about.html

```diff
@@ -0,0 +1,103 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./index.css" />
+    <title>The digital transformation</title>
+  </head>
+  <body>
+    <header>
+      <ul class="social-media">
+        <li class="fb">
+          <a href="#"><img src="./images/facebook.jpg" alt="facebook" /></a>
+        </li>
+        <li>
+          <a href="#"><img src="./images/twitter.png" alt="twitter" /></a>
+        </li>
+        <li><a href="#">English</a></li>
+        <li><a href="#">my page</a></li>
+        <li><a href="#">Logout</a></li>
+      </ul>
+      <div class="menu">
+        <div class="icon">
+          <div class="hamburger">
+            <img src="./images/hamburger.png" id="hamburger" alt="hamburger" />
+            <img src="./images/close.png" alt="close" id="close">
+          </div>
+          <div class="logo">
+            <a href="./"><img src="./images/logo.png" alt="logo" /></a>
+          </div>
+        </div>
+        <ul class="links showlinks" id="links">
+          <li class="link-item"><a href="./about.html">About</a></li>
```

Comment:
- [ ] Great Job making your logo a link to the home page :muscle: . `index.html`, however, I kindly suggest that on the `about.html` page, you change the `about page` link to the home page link as the `about page` link on your `about page` is redundant because it links to the same about page.

![Screenshot from 2022-05-26 11-40-54](https://user-images.githubusercontent.com/52934128/170472366-1bbd384b-f366-44c9-ac72-ee8c19d87d23.png)